doc/project1-StandaloneKV.md

```diff
@@ -1,8 +1,25 @@
 # Project1 StandaloneKV
 
-In this project, you will build a standalone key/value storage [gRPC](https://grpc.io/docs/guides/) service with the support of the column family. Standalone means only a single node, not a distributed system. [Column family]( <https://en.wikipedia.org/wiki/Standard_column_family> ) (it will abbreviate to CF below) is a term like key namespace, namely the values of the same key in different column families is not the same. You can simply regard multiple column families as separate mini databases. It’s used to support the transaction model in the project4, you will know why TinyKV needs the support of CF then.
+In this project, you will build a mini key-value storage service with the following characteristics.
 
-The service supports four basic operations: Put/Delete/Get/Scan. It maintains a simple database of key/value pairs. Keys and values are strings. `Put` replaces the value for a particular key for the specified CF in the database, `Delete` deletes the key's value for the specified CF, `Get` fetches the current value for a key for the specified CF, and `Scan` fetches the current value for a series of keys for the specified CF.
+#### Standalone Key-value storage
+
+Standalone means only a single node, not a distributed system. This StandaloneKV in Project1 is just for storing key/value pairs locally and its underlying storage capability is built upon a pure go key-value database called badgerDB.
```

Comment:
Please use "key-value" consistently.

Reply:
Also "Standalone means only a single node, not a distributed system" sounds in Chinglish. @dcalvin do you have an idea expressing this?